android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,2 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactnativeshadowview">
-
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
why do we need this change?

Reply:
It's not. I think this slipped through, I am sorry.